### DISCUSSION TITLE
Any plans to support sortable selected multiple values?

### DISCUSSION THREAD

**Discussion by mustafamw** [csandman/chakra-react-select#310] [Q&A]
Here's example of sortable values:

![chrome-capture-2024-2-9](https://github.com/csandman/chakra-react-select/assets/8669995/623506c7-cfd7-45f2-842f-a0b3bfaf0035)

https://codesandbox.io/p/sandbox/react-select-sortable-handle-hoc-ipud9?file=%2Fexample.js%3A5%2C21

**Comment by csandman** [comment]
This is something that is implemented with an external package, so you should be able to implement something similar. However, the package [`react-sortable-hoc` is deprecated now](https://github.com/clauderic/react-sortable-hoc), so I'd recommend using a more modern solution like [`dnd-kit`](https://github.com/clauderic/dnd-kit). In fact, there's a proposal to replace the demo in the original `react-select` docs with a `dnd-kit` one: https://github.com/JedWatson/react-select/issues/5159

And someone actually went ahead and made a PR for this as well: https://github.com/JedWatson/react-select/pull/5212

I took a stab at replacing `react-select` in this example with `chakra-react-select`, and it works decently well, but it's not perfect: https://stackblitz.com/edit/vitejs-vite-xrexjh?file=src%2Foriginal-select-sort.tsx,src%2Fmulti-select-sort.tsx,src%2FApp.tsx

It has issues with wrapping displaced options onto a new line, but that could probably be fixed if you looked through [the `dnd-kit` docs](https://docs.dndkit.com/).
